src/Api/ProjectsApi.php

```diff
@@ -75,4 +129,58 @@ public function projectsUserUserIdGet(string $userId, string $maxVersion = null,
   {
     // TODO: Implement projectsUserUserIdGet() method.
   }
+
+  /**
+   * @param Program[] $programs
+   *
+   * @return Project[]
+   */
+  private function getResponseData($programs)
+  {
+    $projects = [];
+    foreach ($programs as &$program)
+    {
+      $result = [
+        'id' => $program->getId(),
```

Comment:
Please just create an instance of the project model -> `vendor/catrobat/capi/Model/Project.php`.  using the ctor